persistence/genesis.go

```diff
@@ -205,119 +205,3 @@ func (p PostgresContext) GetAllPools(height int64) (accs []modules.Account, err
 	}
 	return
 }
-
-func (p PostgresContext) GetAllApps(height int64) (apps []modules.Actor, err error) {
-	ctx, tx, err := p.getCtxAndTx()
-	if err != nil {
-		return nil, err
-	}
-	rows, err := tx.Query(ctx, types.ApplicationActor.GetAllQuery(height))
-	if err != nil {
-		return nil, err
-	}
-	var actors []*types.Actor
-	for rows.Next() {
-		var actor *types.Actor
-		actor, height, err = p.getActorFromRow(rows)
-		if err != nil {
-			return
-		}
-		actors = append(actors, actor)
-	}
-	rows.Close()
-	for _, actor := range actors {
-		actorWithChains, err := p.getChainsForActor(ctx, tx, types.ApplicationActor, actor, height)
-		if err != nil {
-			return nil, err
-		}
-		apps = append(apps, actorWithChains)
-	}
-	return
-}
-
-func (p PostgresContext) GetAllValidators(height int64) (vals []modules.Actor, err error) {
-	ctx, tx, err := p.getCtxAndTx()
-	if err != nil {
-		return nil, err
-	}
-	rows, err := tx.Query(ctx, types.ValidatorActor.GetAllQuery(height))
-	if err != nil {
-		return nil, err
-	}
-	var actors []*types.Actor
-	for rows.Next() {
-		var actor *types.Actor
-		actor, height, err = p.getActorFromRow(rows)
-		if err != nil {
-			return
-		}
-		actors = append(actors, actor)
-	}
-	rows.Close()
-	for _, actor := range actors {
-		actor, err = p.getChainsForActor(ctx, tx, types.ApplicationActor, actor, height)
```

Comment:
I spotted this bug, this should be `ValidatorActor`, fixed in while I moved it to `actor.go`